juniper_official/Solutions/EVPN-VXLAN/vtep-interface-status.rule

```diff
@@ -15,9 +16,16 @@ healthbot {
                 iAgent {
                     file vtep-statistics.yml;
                     table VtepTable;
-                    frequency 30s;
+                    frequency 60s;
                 }
             }
+            field vtep-name {
+                sensor vxlan {
+                    path name;
+                }
+                type string;
+                description "Vtep interface name";
+            }
```

Comment:
Is below correct?
"where "oper-status =~ /+*/";"

Reply:
modified the regex...